### DISCUSSION TITLE
能黑到教员头上，那只能说加油了

### DISCUSSION THREAD

**Discussion by lkr4545plb** [WeNeedHome/SummaryOfLoanSuspension#548] [General]
教员都黑，没救了。到时候血溅我身上也只会拍手称快了

**Comment by lkr4545plb** [comment]
![image](https://user-images.githubusercontent.com/37802834/179173635-9adf4a32-65bd-4c58-b7d6-ffa20ae65664.png)

**Reply by oceanzhang860411** [reply]
教員？沒學過歷史就別噴糞了，毛澤東搞得土改，鬥得就是買得起房的人。

**Reply by lkr4545plb** [reply]
你可真是你妈个小天才，都能黑到土改上![image](https://user-images.githubusercontent.com/37802834/179174810-7bac5fb4-b13d-40da-82d7-2f236d5d6412.png)

**Reply by oceanzhang860411** [reply]
會用github不會用Google是吧？回你的墻裏呆著

**Reply by lkr4545plb** [reply]
去你妈井里呆着，智商为负数的蛙 ，用个Google还有优越感呢？真以为自己懂了啥？

**Reply by MicroCountry** [reply]
用谷歌，繁体字，你就嘚瑟了？蠢货，狗腿子

**Reply by oceanzhang860411** [reply]
無能狂吠

**Reply by lkr4545plb** [reply]
> 無能狂吠

呱呱呱

**Reply by ppinkiris** [reply]
来，跟我念。台巴子死一车亲妈

**Reply by xiejun946** [reply]
> 教員？沒學過歷史就別噴糞了，毛澤東搞得土改，鬥得就是買得起房的人。

台湾狗，滚远点

**Reply by TALK1949** [reply]
现在五毛粉红就这样，看到繁体字谈论中国的问题就跟杀了他们亲妈一样。
弄得我也想切换繁体来上几句😀

**Reply by John-zhan** [reply]
> 来，跟我念。台巴子死一车亲妈

真给共产主义丢脸

**Reply by fnnnnng** [reply]
奴才名字都不会叫，自己给主子封了个教员名称，教你mb

**Comment by program-thinked** [comment]
屠支大佐毛泽东还要黑？几千万人白死了不成。

**Reply by lkr4545plb** [reply]
总共就4亿人 几千万人张口就来 要就放证据  我还说是你爷强了你妈呢！

**Reply by go2006010215** [reply]
1. 延安整风运动
整风运动的实行，使毛泽东成为党主席，在党中央的地位更为确立，被许多学者认为是毛泽东个人崇拜的开端。 该运动造成诸多冤假错案，大量人员遭到迫害，知识分子群体受到巨大冲击。 有研究指该运动，共造成超过一万人死亡。

来源
https://zh.m.wikipedia.org/zh-hans/%E6%95%B4%E9%A2%A8%E9%81%8B%E5%8B%95

2. 反AB团
朱家浩承認红旗社列宁青年社、赣西南政府都有AB团的小组，赣西南AB团的总团长是谢兆元。 於是赣西南特委抓捕谢兆元等人。 當年5月至9月，于都有1000余人被指稱為AB团分子並遭到捕杀，城关区就有近200人被杀。

来源
https://zh.m.wikipedia.org/zh-hans/%E5%8F%8DAB%E5%9C%98

3. 三年困难时期1500万至5500万人的非正常死亡

据各方估计，三年困难时期间发生的大饥荒共造成了中国大陆1500万至5500万人的非正常死亡，被广泛视为人类历史上规模最大的饥荒，也被认为是人类历史上最严重的人为灾难之一

来源
https://zh.wikipedia.org/wiki/%E4%B8%89%E5%B9%B4%E5%9B%B0%E9%9A%BE%E6%97%B6%E6%9C%9F

袁隆平：三年饥荒饿死几千万
https://www.youtube.com/watch?v=EM3vH73tiSg

4. 反右运动

来源
https://zh.wikipedia.org/wiki/%E5%8F%8D%E5%8F%B3%E8%BF%90%E5%8A%A8

5. 文革期间非正常死亡人口达数十万至2000万不等
据各方估计，文革期间中国大陆非正常死亡人口达数十万至2000万不等，从北京的“红八月”起，诸多地区发生了大屠杀，包括广西文革屠杀（及大规模人吃人事件）、内蒙古内人党事件、广东文革屠杀、云南文革屠杀、湖南文革屠杀等，亦爆发了二十世纪世界上最重大的人为技术灾难“河南“75·8”水库溃坝”。
来源
https://zh.wikipedia.org/wiki/%E6%96%87%E5%8C%96%E5%A4%A7%E9%9D%A9%E5%91%BD


统计

林辉：中共70年来杀了多少人？
https://www.epochtimes.com/gb/19/9/25/n11546701.htm

**Reply by hqqlucky** [reply]
> 屠支大佐毛泽东还要黑？几千万人白死了不成。
张口就来几千万，真有几千万还反攻不了大陆，只能说凯升大队长是真滴菜。总共就4亿人，你这几千万就代表着你爷爷奶奶外公外婆家人必然有几个因为这死了。这还不加入果党？

**Reply by zilazhu** [reply]
> 

波儿，我刚刚在天安门玻璃棺材前替你向腊肉许愿了，毛腊肉告诉我他以你全家户籍为担保，保你家顺风顺水出人头地，你要是成了人上人要记得去给他还愿。

**Reply by John-zhan** [reply]
> > 屠支大佐毛泽东还要黑？几千万人白死了不成。
> > 张口就来几千万，真有几千万还反攻不了大陆，只能说凯升大队长是真滴菜。总共就4亿人，你这几千万就代表着你爷爷奶奶外公外婆家人必然有几个因为这死了。这还不加入果党？

首先你这里有幸存者偏差问题，其次你可以使用明文的，为啥非用谐音字。

**Reply by TALK1949** [reply]
> > 张口就来几千万，真有几千万还反攻不了大陆，只能说凯升大队长是真滴菜。总共就4亿人，你这几千万就代表着你爷爷奶奶外公外婆家人必然有几个因为这死了。这还不加入果党？

再给这位普及一下历史知识，当年中国共产党扶持的柬埔寨共产政权“红色高棉”屠杀了150万至300万人，约占当时柬埔寨总人口的四分之一，张口给你来一下？

**Reply by maomaoc474** [reply]
这种所以傻逼数据 都出自一方之口 谁信就是傻逼。
事实是 美洲土著 澳洲土著 非洲大部分部落 生物上彻底灭绝了。。。。
印度 东南亚 非洲大部分地区 文化上灭绝了 改说英语了。。。。
你他妈 跟我扯几百万 几千万， 张嘴胡编，你就编它是几亿，它怎么就不至于灭族呢！加拿大 美国教堂下的儿童尸骨在瑟瑟发抖！！！

**Reply by liulangderen-11** [reply]
@lkr4545plb 把无能狂吠四个字体现的淋漓尽致。别人这么辛苦的给你贴出证据，怎么连谢谢都不会说？怎么不继续狂吠了？

**Reply by liulangderen-11** [reply]
> 这种所以傻逼数据 都出自一方之口 谁信就是傻逼。 事实是 美洲土著 澳洲土著 非洲大部分部落 生物上彻底灭绝了。。。。 印度 东南亚 非洲大部分地区 文化上灭绝了 改说英语了。。。。 你他妈 跟我扯几百万 几千万， 张嘴胡编，你就编它是几亿，它怎么就不至于灭族呢！加拿大 美国教堂下的儿童尸骨在瑟瑟发抖！！！

所以你是哪国人，自己国家的事不关心，倒是对别人的历史挺积极。以你的逻辑，别人告诉你妈被人打死了，你上蹿下跳说隔壁老王也被人打死了，不知道你要表达什么。

**Comment by Accelerator6666** [comment]
三年自然灾害死的人真的可以上千万。。

**Reply by toxin08x** [reply]
我不知道其他人怎么想，做出这种饿死这么多人的政府（咱按少的算，就算几十万几百万），无论它再好，再怎么反思（现在甚至一点反思都没有），我也完全无法原谅。这就是我润出来的原因之一。

**Reply by Accelerator6666** [reply]
> 我不知道其他人怎么想，做出这种饿死这么多人的政府（咱按少的算，就算几十万几百万），无论它再好，再怎么反思（现在甚至一点反思都没有），我也完全无法原谅。这就是我润出来的原因之一。

怎么说呢，当时老一辈都说饥荒的时候还不允许他们出去讨生活。。。这个操作是真的谜一样，都已经饥荒了，人都快饿死了，还不允许出去讨生活，令人发指

**Reply by TALK1949** [reply]
> 怎么说呢，当时老一辈都说饥荒的时候还不允许他们出去讨生活。。。这个操作是真的谜一样，都已经饥荒了，人都快饿死了，还不允许出去讨生活，令人发指

难以理解？看看前阵子上海。

**Reply by JayChou55** [reply]
> > 我不知道其他人怎么想，做出这种饿死这么多人的政府（咱按少的算，就算几十万几百万），无论它再好，再怎么反思（现在甚至一点反思都没有），我也完全无法原谅。这就是我润出来的原因之一。
> 
> 怎么说呢，当时老一辈都说饥荒的时候还不允许他们出去讨生活。。。这个操作是真的谜一样，都已经饥荒了，人都快饿死了，还不允许出去讨生活，令人发指

你以为户籍这政策的目的是什么 你以为呢

**Reply by xiejun946** [reply]
有什么意义，土地改革，冤假错案，哪个国家朝代没有，历史的车轮滚滚，死了多少人，这些都是历史发展道路中不可避免的牺牲，从结果上来说，中国养活了占世界1/6的人口，世界第二大经济体，大国军事，经济，这些都是有意义的，如果说预售制的话，也是为了中国发展不可避免的，有利于国家发展。

**Reply by ukindy** [reply]
> 有什么意义，土地改革，冤假错案，哪个国家朝代没有，历史的车轮滚滚，死了多少人，这些都是历史发展道路中不可避免的牺牲，从结果上来说，中国养活了占世界1/6的人口，世界第二大经济体，大国军事，经济，这些都是有意义的，如果说预售制的话，也是为了中国发展不可避免的，有利于国家发展。

是的，你我皆汽油

**Reply by mand-byte** [reply]
> 有什么意义，土地改革，冤假错案，哪个国家朝代没有，历史的车轮滚滚，死了多少人，这些都是历史发展道路中不可避免的牺牲，从结果上来说，中国养活了占世界1/6的人口，世界第二大经济体，大国军事，经济，这些都是有意义的，如果说预售制的话，也是为了中国发展不可避免的，有利于国家发展。

又开始比烂了。中国的社会进步就靠比烂了。

**Reply by oceanzhang860411** [reply]
> 有什么意义，土地改革，冤假错案，哪个国家朝代没有，历史的车轮滚滚，死了多少人，这些都是历史发展道路中不可避免的牺牲，从结果上来说，中国养活了占世界1/6的人口，世界第二大经济体，大国军事，经济，这些都是有意义的，如果说预售制的话，也是为了中国发展不可避免的，有利于国家发展。

果然是蛆的思維，人家100年前吃過屎，你現在就一定要在把屎的香噴噴是吧？人家全民醫保，怎麼不學，人家可以公開議論執政黨領導人，怎麼不學？

**Reply by charliechaw** [reply]
你去牺牲吧，几千万人命被你轻轻带过，你真的一点对同胞的同情心都没有啊> 有什么意义，土地改革，冤假错案，哪个国家朝代没有，历史的车轮滚滚，死了多少人，这些都是历史发展道路中不可避免的牺牲，从结果上来说，中国养活了占世界1/6的人口，世界第二大经济体，大国军事，经济，这些都是有意义的，如果说预售制的话，也是为了中国发展不可避免的，有利于国家发展。

**Reply by mand-byte** [reply]
中国是一个什么样的机器或是人？它是如何养了1/6世界人口的？你吃饭的钱是中国这个实体免费给你的？
什么叫不可避免？中国几千年的独裁专政有从中吸取经验吗？到现在还是重复。

**Reply by liulangderen-11** [reply]
> 有什么意义，土地改革，冤假错案，哪个国家朝代没有，历史的车轮滚滚，死了多少人，这些都是历史发展道路中不可避免的牺牲，从结果上来说，中国养活了占世界1/6的人口，世界第二大经济体，大国军事，经济，这些都是有意义的，如果说预售制的话，也是为了中国发展不可避免的，有利于国家发展。

你所谓“养活了占世界1/6的人口，世界第二大经济体”的经济成就，是在“土改”“冤假错案”的制造者腊肉死掉之后，通过确立市场经济之后才做到的。不抛弃土改的路子，你自诩的辉煌经济能实现？还是说沿着“土改”的路子，经济能腾飞？你怕不是不知道土改是干什么的吧？土改是取消私有制的，而经济建设是必须承认私有制的。你所谓的“有什么意义”的取消私有制，压根就达不到你引以为豪的“经济体”。

**Comment by ineedabitchlikeu** [comment]
3年灾害完全是人祸 甩锅一流

**Comment by cc2218** [comment]
个人崇拜就是他打下的基础。您的发言就是他魔咒的结果。这个魔咒，是中华民族更上一层的障碍。
毛有进步，但是功过要分开。

**Reply by GitHub23333** [reply]
自发崇拜是他打下的基础？东方红是谁编的歌？

**Comment by oceanzhang860411** [comment]
土改，斗的就是擁有土地房產的人。傻子還擁護毛臘肉，看來你是欠圖了
土地改革法的第二条规定：“没收地主的土地、耕畜、农具、多余的粮食及其在农村中多余的房屋……”。这就是说，这些土地财产在法律上已经不是地主所有的了。在土地改革之前，地主对这些财产，只有保护之责，而无任何支配之权，更不得破坏、分散、出卖和转移。否则，就是对国家人民财产的损害，就是犯法，就必须受到国家法律的制裁。关于这一点，人民政府早有明文规定。土地改革法第八条具体地规定：“本法规定所有应加没收和征收的土地，在当地解放以后，如以出卖、出典、赠送或其他方式转移分散者，一律无效。此项土地，应计入分配土地的数目之内。”又在第三十三条中规定：“在土地改革完成以前，为保证土地改革的秩序及保护人民的财富，严禁一切非法的宰杀耕畜，砍伐树木，并严禁荒废土地，破坏农具、水利、建筑物、农作物或其他物品，违者应受人民法庭的审判及处分。”因此，对于那些藐视人民政府法令，明知故犯的不法地主，决不能再予宽容，必须依法制裁。
http://www.nlc.cn/dsb_zt/xzzt/ghgdsj/1950/event11/wzbd47/201912/t20191211_185070.htm

**Reply by oceanzhang860411** [reply]
點踩的是什麼太監？你國自己發布的報紙，這都點踩，反革命嗎？

**Reply by naifenmizuha** [reply]
祝你穿越到民国当无地农民

**Reply by oceanzhang860411** [reply]
什麼，你現在有地？窪地不是只有使用權？要不要自己讀一下下面這個法律？
[中华人民共和国城镇国有土地使用权出让和转让暂行条例](http://www.gov.cn/zhengce/2020-12/25/content_5574204.htm)

**Reply by TALK1949** [reply]
> 祝你穿越到民国当无地农民

民国不需要穿越，去中华民国台湾就行了。🤔

**Comment by 64xi** [comment]
教员😭你快回来图纸吧

**Reply by dagoohan** [reply]
膠原😭😭😭我的膠原😭😭😭

**Comment by Kasianian** [comment]
教员？宣扬毛泽东的功，甚至把毛泽东的尸体放在天安门前是共产党维持其独裁政权合法性的唯一牌面。 可不要忘了，如果不是毛泽东还活着，文革就会继续屠杀知识分子并摧残中国的历史遗迹，只有毛泽东死了出现权力真空，中共政治局常委才有机会铲除四人帮夺权，简直暗无天日的历史事件，粉红还能继续吹教员？

**Reply by Kasianian** [reply]
关于毛的评价，评价他不应该看他被宣传的理想有多伟大，而是看他为人民实际做了什么？他军事谋略确实没得说，土匪性质般抢夺地主几代人积累的土地财富，以土地为引诱，让农民充当燃料为他打天下并赶跑了国民政府，最後是违背初衷让共产党实现了土地兼并，经济治理方面更是一团糟，民生方面步步灾难反人类，口口声声说蒋介石是独裁者想当皇帝，晚年通过自身影响力搞文革搞个人崇拜当上了皇帝（实际比皇帝的权力还要大），文革期间掌握实权，没人敢说他的不是，红卫兵看到你家的毛泽东的头像歪了脏了那都是得抄家批斗之死

**Reply by icm-ai** [reply]
天下不都是打易治难么，相对来说？

**Reply by yangyuan-zhen** [reply]
正确的

**Comment by TALK1949** [comment]
> 总共就4亿人 几千万人张口就来 要就放证据 我还说是你爷强了你妈呢！

https://github.com/WeNeedHome/SummaryOfLoanSuspension/discussions/548#discussioncomment-3153942

我奉劝那些想说这话的人说话前先过过脑子，真实的历史远比教科书里装点的残酷得多，当然，如果真的了解历史也不会说出这种无知的话，虽然说这也不能全怪他们，毕竟我们从小到大就是这么被教育过来的，不过我们那代粉红真没有这么无知，可见这些年在教育和宣传上，没少下功夫，我知道有些人很不愿意听“洗脑”这个词，但这恰恰就是。

![image](https://user-images.githubusercontent.com/39082705/180428500-828acb27-5e22-4b7a-8efe-b046f8f74a40.png)
https://youtu.be/3_1vCE3rp9Q

注：本人表达的观点皆与“政治”无关，只就问题本身提供观点🐶

[中国共产党政治运动列表](https://zh.m.wikipedia.org/zh-sg/%E4%B8%AD%E5%9B%BD%E5%85%B1%E4%BA%A7%E5%85%9A%E6%94%BF%E6%B2%BB%E8%BF%90%E5%8A%A8%E5%88%97%E8%A1%A8) [来源WIKI]

-------------------中华苏维埃共和国

反AB团	1930年-1931年
	死亡人数7.6万	其中，红二十军和红四军冲突，导致红二十军副排级以上军官全部被杀，史称富田事变

肃清社会民主党	1931年-1931年
	死亡人数6352	起因是有人错喊社会民主党的口号

肃清国民党改组派	1931年-1932年
	死亡人数2万多	湘鄂西苏区肃反4次，杀2万多人；鄂豫皖苏区将200多知识分子肃成5、6人。

-------------------陕甘宁边区

延安整风	1942年-1945年
	死亡人数超过1万	

土地改革运动	1946年-1953年
	死亡人数83万至500万，以土改前地主占农村总人口4.75%（土地38.26%），土改后约占2.6%（土地2.2%）计算，前后减少约667万人。有约5%的富农，部分视同地主处理。 

城工部事件
	1948年-1948年
	死亡人数127，超过3000名城工部党员被开除党籍，在党内起重要作用的干部被大量错杀，严重破坏了闽浙赣省委的地下工作。

-------------------毛泽东时期

镇压反革命运动
	1950年-1953年
	死亡人数，反革命：71.2万或87.46万，土匪：240万

三反五反运动
	1951年-1952年
	中国资产阶级1320人自杀51人遭枪毙，王朝彬著《三反实录》：312万人被调查，123万人被指控贪污，运动共处分39.4万人，枪毙51人。中组部副部长安子文报告称383.6万人被调查，120.3万人被控贪污，至1952年底27万人被处分

胡风反革命集团案
	1952年-1956年
	逮捕92人，查处100万人

肃清暗藏的反革命分子运动
	1955年-1955年
	死亡人数5.3万		

反右运动
	1957年-1958年
	死亡人数4117，55万至460万人被定为右派。夹边沟有3000多名右派因饥饿而死，全国范围内的右派死亡数量未知。
		
大跃进
	1958年-1961年
	死亡人数1500万-5500万，农村供应中断。

四清五反运动
	1962年-1966年
	死亡人数77560，“社会主义教育运动”的一部分

无产阶级文化大革命
	1966年-1977年
	1亿人受迫害，172万至2000万人死亡，703万馀人伤残

清查“五·一六反革命集团”运动
	1967年-1974年
	死亡人数10万，1970年大规模展开
				
一打三反运动	1970年-1972年
	死亡人数15万至20万，周恩来下令按1‰肃清阶级敌人，处决者少，自杀者多。

**Reply by itsoursduty** [reply]
堪称图纸大佐

**Reply by xiejun946** [reply]
别提洗脑这词了，日本教科书有的洗？台湾岛教科书有的洗，不都是当权者修改了，你能保证你的思想没有被别人所影响？提洗脑，就唯心了。说内战死了几千万人，算个屁，美国内战死了62万，哪个国家政权斗争过程中没死人

**Reply by forgor9** [reply]
我算了算这他妈的一共危害了几亿人，你意思是说中国人死了一小半？编也编 的像话一点好吧

**Reply by oceanzhang860411** [reply]
> 我算了算这他妈的一共危害了几亿人，你意思是说中国人死了一小半？编也编 的像话一点好吧

那你解釋下，為什麼公安洩漏的數據只有10億？另外4億人是活在1970年代的紙質戶籍數據上？

**Reply by maomaoc474** [reply]
解释下 台湾地区将秃驴震惊世界的白色恐怖政权 台湾地区打破人类记录的世界最长独裁宵禁军统政府 直到近几年才被叙利亚反超。简直是中华民族的耻辱！台湾谈民主 就是婊子谈处女！

**Reply by liulangderen-11** [reply]
> 别提洗脑这词了，日本教科书有的洗？台湾岛教科书有的洗，不都是当权者修改了，你能保证你的思想没有被别人所影响？提洗脑，就唯心了。说内战死了几千万人，算个屁，美国内战死了62万，哪个国家政权斗争过程中没死人

首先，人家只是陈述事实，你不同意可以列举你的事实，但是你那种“别的国家也洗，所以我们洗也合理”的逻辑真的有够认知下限。
其次，“美国内战死了62万”，按杀人的数量和时间的跨越，跟中国相比绝对是不值一提。你搜遍全网终于捏到人家一点小问题拿来跟你房间里的大象做对比，好意思吗？

**Reply by liulangderen-11** [reply]
> 我算了算这他妈的一共危害了几亿人，你意思是说中国人死了一小半？编也编 的像话一点好吧

怎么，中国人那几十年不能生育，被你给上环了？
恰恰相反，中国人那时候就跟吃了春药的牲口一样能生，以至于刹不住车，后来实行了计划生育。所以这数字不能合理吗？

**Comment by GitHub23333** [comment]
这贴目田度很浓，不过也不奇怪

**Reply by oceanzhang860411** [reply]
> 这贴目田度很浓，不过也不奇怪

我支持共產黨對你天天進行核算檢測

**Comment by John-zhan** [comment]
> 教员都黑，没救了。到时候血溅我身上也只会拍手称快了

不怕有问题，就怕有问题，你玻璃心接受不了问题现状。不解决问题，反手却因为面子上挂不住，去解决提出问题的人。这样以后谁敢提出问题，问题你要认为不存在可以不理呀。别人刚提出问题，你自己觉得脸上挂不住，直接把别人口封了。就这样普通人怎么去拿数据，恐怖一点的讲，记者都不放过啊。

**Comment by fnnnnng** [comment]
什么垃圾东西，还他妈教员，你家的教员才是他，教你mb员，跪久了就好好跪着别出来叫。

**Comment by volans19** [comment]
贱种一个，鉴定完毕，10亿数据不是白泄露的，有种就留着贴别删

**Comment by victorchen276** [comment]
毛泽东还用黑？

**Comment by MAGAWOLA** [comment]
毛泽东就是苏联有毒共产主义思潮下的产物，这些异族思潮真的贻害万年，因国共内战同胞相互残杀多达500万，直至今天共产党还要挤压台湾人民的生存空间，朝鲜战争中美国为避免红色赤化扩展而不得不与红色中国对抗，毛泽东晚年对权力的迷恋，竟然效仿斯大林对党内党外进行大清洗，大量被革了命的冤魂无人追悼，反而有粉红产物不断吹嘘歌颂始作俑者，民族被这些红色赤化的人利用，民族走向了错误发展道路，方向是错的，做的任何纠正都将是徒劳，只不过需要後代不断承受这些苦难。可以说毛泽东是中国上下五千年历史里给中国人民带来最多苦难的人物，他的遗体还在天安门广场水晶棺中，清算他只是时间问题，不以粉红意志为转移。

**Comment by Pacio** [comment]
腊肉有功，但是过太大，折算下来肯定是民族罪人

**Reply by Pacio** [reply]
不对，腊肉可能内心深处也并没有把自己当中国人，所以应该就是纯纯的反人类

**Reply by JayChou55** [reply]
我个人揣测他心是好的期望也是好的，但是 一切都随着 自己儿子在朝鲜战场上飞灰湮灭后产生了变化。。。。这也就能解释为啥建国后，开始不对劲了

**Reply by JayChou55** [reply]
这也是为啥彭德怀后来被搞得一塌糊涂的原因之一，明明让自己儿子去镀铬金，你却把我儿玩死了

**Comment by JayChou55** [comment]
一天挂在广场纪念一天就没希望

**Comment by Cegazhos** [comment]
毛确实有很多错误，当初要是坚持把岛打下来 楼上的台湾乡巴佬就不会这么可怜了

**Reply by JayChou55** [reply]
煞笔 打台湾对你有什么好处。。。穷逼

**Reply by oceanzhang860411** [reply]
> 毛确实有很多错误，当初要是坚持把岛打下来 楼上的台湾乡巴佬就不会这么可怜了

你有資格說別人鄉巴佬？你有免費醫療嗎？你見過選票嗎？習包子經過你家小區你敢開窗？當奴隸還真是會上癮哦

**Reply by Cegazhos** [reply]
> 煞笔 打台湾对你有什么好处。。。穷逼

你好哇 台湾乡巴佬

**Reply by Cegazhos** [reply]
> > 毛确实有很多错误，当初要是坚持把岛打下来 楼上的台湾乡巴佬就不会这么可怜了
> 
> 你有資格說別人鄉巴佬？你有免費醫療嗎？你見過選票嗎？習包子經過你家小區你敢開窗？當奴隸還真是會上癮哦

不打自招了吧 可怜虫

**Reply by oceanzhang860411** [reply]
> > > 毛确实有很多错误，当初要是坚持把岛打下来 楼上的台湾乡巴佬就不会这么可怜了
> > 
> > 
> > 你有資格說別人鄉巴佬？你有免費醫療嗎？你見過選票嗎？習包子經過你家小區你敢開窗？當奴隸還真是會上癮哦
> 
> 不打自招了吧 可怜虫

“计算机信息网络直接进行国际联网，必须使用邮电部国家公用电信网提供的国际出入口信道。任何单位和个人不得自行建立或者使用其他信道进行国际联网。
----《中华人民共和国计算机信息网络国际联网管理暂行规定》

你的主子有允許你出來嗎？

**Reply by Cegazhos** [reply]
> > > > 毛确实有很多错误，当初要是坚持把岛打下来 楼上的台湾乡巴佬就不会这么可怜了
> > > 
> > > 
> > > 你有資格說別人鄉巴佬？你有免費醫療嗎？你見過選票嗎？習包子經過你家小區你敢開窗？當奴隸還真是會上癮哦
> > 
> > 
> > 不打自招了吧 可怜虫
> 
> “计算机信息网络直接进行国际联网，必须使用邮电部国家公用电信网提供的国际出入口信道。任何单位和个人不得自行建立或者使用其他信道进行国际联网。 ----《中华人民共和国计算机信息网络国际联网管理暂行规定》
> 
> 你的主子有允許你出來嗎？

kawaii🥰

**Reply by fnnnnng** [reply]
你先定个小目标：明天不用做核酸

**Reply by Cegazhos** [reply]
> 你先定个小目标：明天不用做核酸

你连你爹在不在高风险地区都不知道就在那意淫？你跟你妈学的意淫自我高潮？

**Reply by fnnnnng** [reply]
> > 你先定个小目标：明天不用做核酸
> 
> 你连你爹在不在高风险地区都不知道就在那意淫？你跟你妈学的意淫自我高潮？

急了急了

**Comment by luowenfeng** [comment]
你有什么资格喊教员? 你是波尔布特还是金日成?

**Comment by xijinping0** [comment]
一讲到毛泽东，真是突开沼气池了，一个个激动的。

[![突开](https://img.youtube.com/vi/B8V4rrKuTkE/0.jpg)](https://youtu.be/B8V4rrKuTkE?t=113)

https://github.com/xijinping0
《[习近平与他的情人们](https://lovers.xijinping.one)》

**Comment by liyan0628** [comment]
来信已收到，谢谢！

**Comment by Hcreak** [comment]
孝死了 如丧考妣了

**Reply by Sunovn** [reply]
7年来第一次遇到同头像的😂
